web/src/pages/NotFoundPage.tsx

```
@@ -0,0 +1,56 @@
+            active:shadow-neu-pressed active:scale-[0.98]
+            transition-all duration-200
+          "
+        >
+          <FiHome className="text-lg group-hover:scale-110 transition-transform" />
+          <span>Return to Base</span>
+        </button>
+      </motion.div>
+
+      {/* Footer Decoration */}
+      <div className="absolute bottom-8 text-[10px] text-text-secondary/30 font-mono uppercase tracking-[0.3em]">
+        NYX Secure Protocol // End of Line
```

Biome (2.4.6)

[error] Wrap comments inside children within braces.

(lint/suspicious/noCommentText)